### ISSUE TITLE
Add telemetry

### ISSUE THREAD

**Issue by dirtybirthdaycake** [spacialsec/SadCamper#1]
I'm interested how many people are using SadCamper. Perhaps you could add a telemetry call as the first order of business logic to SadCamper?

**Comment by spacialsec** [comment]
Good idea @dirtybirthdaycake! We believe @Karma-X-Inc simply greps their apache logs for requests to "/hc_usage".  we will do the same for SadCamper!
